open-source-software-submissions/intro.md

```diff
@@ -2,3 +2,6 @@
 
 These are a collection of guides for the peer review and submission process.
 They are broken down by the various roles that carry out a submission.
+
+```{tableofcontents}
```

Comment:
cool - what does this do @choldgraf ? `{tableofcontents}`

Reply:
it just adds in a table of contents for the `sections` that are below the current page